php/libraries/Candidate.class.inc

```diff
@@ -193,7 +193,7 @@ class Candidate implements \LORIS\StudyEntities\AccessibleResource,
      * it is an optional parameter based on configuration
      * settings. However, this will not be backward compatiable.
      *
-     * @param int         $centerID              centerID of the site to which the
+     * @param CenterID    $centerID              centerID of the site to which the
```

Comment:
in line 102 missing "
            case 'RegistrationCenterID':
                $this->candidateInfo[$key] = new \CenterID($value);
                break;
"

Reply:
It is probably that RegistrationCenterID can't be found by test case, so the timepoint runs pselectRow again.